layouts/index.html

```diff
@@ -10,7 +10,7 @@
       <section id="why">
         <div class="content-center mb2">
           <h2>The web of tomorrow needs IPFS today</h2>
-          <h5>IPFS aims to surpass HTTP in order to build a better web for all of us.</h5>
+          <h5>IPFS aims to surpass HTTP, helping build a better web for all of us.</h5>
```

Comment:
`helping` feels less active participant or foundational. Maybe just simplify further: `surpass HTTP, to build a better web`

Separate: I don't think `surpass` is right either, but we don't need to fix that here. Something to stew on.

Reply:
Done.